src/app/components/BibPage/BibMainInfo.jsx

```diff
@@ -1,34 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router';
 import { isEmpty as _isEmpty } from 'underscore';
 
 const BibMainInfo = ({ bib = {} }) => {
   if (_isEmpty(bib)) return null;
 
-  const title = bib.title && bib.title.length ? bib.title[0] : '';
   const materialType = bib && bib.materialType && bib.materialType[0] ?
     bib.materialType[0].prefLabel : null;
+  const author = bib.creatorLiteral && bib.creatorLiteral.length ?
+    `${bib.creatorLiteral[0]}` : '';
+  const contributor = bib.contributorLiteral && bib.contributorLiteral.length ?
+    bib.contributorLiteral : null;
+  const yearPublished = bib && bib.dateStartYear ? bib.dateStartYear : null;
+  const placeOfPublication = bib && bib.placeOfPublication ? bib.placeOfPublication : null;
+  const publisher = bib && bib.publisher ? bib.publisher : null;
   const language = bib && bib.language && bib.language[0] ?
     bib.language[0].prefLabel : null;
-  const yearPublished = bib && bib.dateStartYear ? bib.dateStartYear : null;
-  const usageType = bib && bib.actionType && bib.actionType[0] ?
-    bib.actionType[0].prefLabel : null;
 
   return (
-    <div className="nypl-item-details">
-      <h1>{title}</h1>
-      <div className="nypl-item-info">
-        <p>
-          <span className="nypl-item-media">{materialType}</span>
-          {language && ` in ${language}`}
-        </p>
-        <p>{bib.extent} {bib.dimensions}</p>
-        <p>
-          {bib.placeOfPublication} {bib.publisher} {yearPublished}
-        </p>
-        <p className="nypl-item-use">{usageType}</p>
-      </div>
-    </div>
+    <span>
+      <dt>FORMAT</dt>
+      <dd>
+        {materialType}
+        {language && ` in ${language}`}
+      </dd>
+      <dt>AUTHOR</dt>
```

Comment:
You should add some logic here to not display the term if the definition is not there, example:
![screen shot 2017-07-05 at 3 40 55 pm](https://user-images.githubusercontent.com/1280564/27881886-5b993e76-6198-11e7-98cb-d2a01f1152d8.png)

I think it should be fixed if you do the suggestion above.